libz-rs-sys-cdylib/zpipe.c

```diff
@@ -177,13 +198,15 @@ int main(int argc, char **argv)
 {
     int ret;
 
+    fputs("what the hell\n", stderr);
+
     /* avoid end-of-line conversions */
     SET_BINARY_MODE(stdin);
     SET_BINARY_MODE(stdout);
 
     /* do compression if no arguments */
     if (argc == 1) {
-        ret = def(stdin, stdout, Z_DEFAULT_COMPRESSION);
+        ret = def(stdin, stdout, Z_BEST_SPEED);
```

Comment:
Why was this changed?

Reply:
no idea how that happened, I'll change that back